volatility3/framework/interfaces/objects.py

```diff
@@ -169,6 +178,11 @@ def cast(self, new_type_name: str, **additional) -> 'ObjectInterface':
                                         size = object_template.size)
         return object_template(context = self._context, object_info = object_info)
 
+    def at_layer(self, new_layer_name) -> 'ObjectInterface':
```

Comment:
This is a highly specific method, you can't change the offset when you're doing it and it also doesn't offer any control over `native_layer_name` which is used to allow pointers to point from one layer to another.  This isn't going to make it in.

Reply:
removed it